package-lock.json

```diff
@@ -3582,7 +3582,6 @@
       "resolved": "https://registry.npmjs.org/fsevents/-/fsevents-1.1.3.tgz",
       "integrity": "sha512-WIr7iDkdmdbxu/Gh6eKEZJL6KPE74/5MEsf2whTOFNxbIoIixogroLdKYqB6FDav4Wavh/lZdzzd3b2KxIXC5Q==",
       "dev": true,
-      "optional": true,
```

Comment:
Probably remove the package-lock.json from the PR since the package.json is unchanged.